Formula/d/deno.rb

```diff
@@ -40,22 +41,22 @@ class Deno < Formula
   # TODO: Remove this and `v8` resource when https://github.com/denoland/rusty_v8/issues/1065 is resolved
   # VERSION=#{version} && curl -s https://raw.githubusercontent.com/denoland/deno/v$VERSION/Cargo.lock | grep -C 1 'name = "v8"'
   resource "rusty_v8" do
-    url "https://static.crates.io/crates/v8/v8-0.106.0.crate"
-    sha256 "a381badc47c6f15acb5fe0b5b40234162349ed9d4e4fd7c83a7f5547c0fc69c5"
+    url "https://static.crates.io/crates/v8/v8-130.0.1.crate"
```

Comment:
raise an upstream issue about this version scheme change, https://github.com/denoland/rusty_v8/issues/1661